server/apps/publish/formatters/aap_sms_formatter.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from apps.publish.formatters import Formatter
+import superdesk
+from bs4 import BeautifulSoup
+from superdesk.errors import FormatterError
+
+
+class AAPSMSFormatter(Formatter):
+    def format(self, article, subscriber):
+        """
+        Constructs a dictionary that represents the parameters passed to the SMS InsertAlerts stored procedure
```

Comment:
Yes, I intend to change the can_format to test for duplicates, for that it will need the _id